benchmark.sh

```diff
@@ -0,0 +1,10 @@
+if [ ! -f ./interpreter-c ] && [ $# -lt 1 ]; then
```

Comment:
I find this check slightly odd. If the interpreter is not built I will see an message about needing an argument.

I would just assume the interpreter is built to be honest.